src/main/java/codeu/controller/ActivityFeedServlet.java

```diff
@@ -64,7 +69,36 @@ void setMessageStore(MessageStore messageStore) {
     this.messageStore = messageStore;
   }
 
-  /**
+
+  public static class DateFormatter {
+
+    /**
+     * Utility function to convert java Date to TimeZone format
+     *
+     * @param date
```

Comment:
nit: if using the javadoc @params, also include a blurb about what the param is
i.e. @param date date to be formatted

Reply:
Done